fendermint/app/src/main.rs

```diff
@@ -14,14 +23,7 @@ async fn main() {
 
     // Log events to stdout.
     if let Some(level) = opts.tracing_level() {
-        // Writing to stderr so if we have output like JSON then we can pipe it to something else.
-        let subscriber = FmtSubscriber::builder()
-            .with_max_level(level)
-            .with_writer(std::io::stderr)
-            .finish();
-
-        tracing::subscriber::set_global_default(subscriber)
-            .expect("setting default subscriber failed");
+        create_log(level, &opts.home_dir).expect("cannot create logging");
```

Comment:
I think we should add a setting to configure the output path, and if we want the log to be output on std err/std output, or to a file.

Reply:
I think it's right to do both. I had a [similar thing in my Scala programs](https://github.com/aakoshh/metronome/blob/e5b657b0ac68fe3bf37ee0c11fb8f6f55ea0e7ec/metronome/examples/resources/logback.xml#L7-L40) except I was able to configure them through XML, so we can enable the debug level on a package-by-package basis (third parties as well) without recompiling.